supports/tools/docker/multihost/docker-compose.yaml

```diff
@@ -4,7 +4,6 @@ services:
   hadoop-namenode:
     image: cloud-hub.adsw.io/library/hadoop-namenode:${HADOOP_VERSION:-3.3.6}
     hostname: hadoop-namenode.demo
-    container_name: hadoop-namenode
```

Comment:
Removing container names will cause the Kerberized HDFS cluster to fail due to the use of the `_HOST` variable in the configurations (both SSM and HDFS sites). We have only 1 instance of each service, so we can try to inline the hostnames and check if it works